script/script.js

```diff
@@ -1,87 +1,264 @@
+          url3: "images/7.jpeg",
+          url4: "images/6.jpeg",
+          url5: "images/4.jpeg",
+          url6: "images/5.jpeg",
+          url7: "images/2.jpeg"
+        }
+      ]
+    },
+    {
+      url: "images/8.jpeg",
+      related: [
         {
-            "url": "images/4.jpeg",
-        },
+          url1: "images/8.jpeg",
+          url2: "images/1.jpeg",
+          url3: "images/7.jpeg",
+          url4: "images/6.jpeg",
+          url5: "images/4.jpeg",
+          url6: "images/5.jpeg",
+          url7: "images/2.jpeg"
+        }
+      ]
+    },
+    {
+      url: "images/9.jpeg",
+      related: [
         {
-            "url": "images/5.jpeg",
-        },
+          url1: "images/3.jpeg",
+          url2: "images/1.jpeg",
+          url3: "images/7.jpeg",
+          url4: "images/6.jpeg",
+          url5: "images/4.jpeg",
+          url6: "images/5.jpeg",
+          url7: "images/2.jpeg"
+        }
+      ]
+    },
+    {
+      url: "images/10.jpeg",
+      related: [
         {
-            "url": "images/6.jpeg",
+          url1: "images/3.jpeg",
+          url2: "images/1.jpeg",
+          url3: "images/7.jpeg",
+          url4: "images/6.jpeg",
+          url5: "images/4.jpeg",
+          url6: "images/5.jpeg",
+          url7: "images/2.jpeg"
         }
-    ]
+      ]
+    },
+    {
+      url: "images/1.jpeg"
+    },
+    {
+      url: "images/2.jpeg"
+    },
+    {
+      url: "images/3.jpeg"
+    },
+    {
+      url: "images/4.jpeg"
+    },
+    {
+      url: "images/5.jpeg"
+    },
+    {
+      url: "images/6.jpeg"
+    },
+    {
+      url: "images/7.jpeg"
+    },
+    {
+      url: "images/8.jpeg"
+    },
+    {
+      url: "images/9.jpeg"
+    },
+    {
+      url: "images/10.jpeg"
+    },
+    {
+      url: "images/1.jpeg"
+    },
+    {
+      url: "images/2.jpeg"
+    },
+    {
+      url: "images/3.jpeg"
+    },
+    {
+      url: "images/4.jpeg"
+    },
+    {
+      url: "images/5.jpeg"
+    },
+    {
+      url: "images/6.jpeg"
+    }
+  ]
 };
-$(document).ready(function(){
-    $.each(data.items, function (i, f) {
-        $("ul").append("<li><a href=''><img src='" + f.url +  " '</a></li>");
+
+$(document).ready(function() {
+  // populate the gallery item
+  $.each(data.items, function(i, f) {
+    $("ul").append(
+      "<li class='gallery-item cp' id='" +(i + 1) +"'><a><img  src='" +f.url +" '</a></li>"
+    );
+  });
+
+  var modal = document.getElementById("myModal");
+
+  $(".gallery-item").on("click", function() {
+    $(".modal").addClass("block");
+    var pos = $(this).attr("id");
+    var index = data.items[pos - 1];
+    // navigate to next slide
+    function nextSlide() {
+      var $active = $('.my-slides.fade.block');
+      var $slides = $('.my-slides');  
+      var $next = $active.data('attr') === "last" ? $slides.first() : $active.next();
+      $next.addClass('block');
+      $active.removeClass('block');
+    }
+    // navigate to previous slide
+    function prevSlide(){
```

Comment:
use a single function for both next and prev

Reply:
moved the json to a separate json file